model/Core/Classes/Actor.md

```diff
@@ -18,3 +18,7 @@ Actor is a grouping of identifying characteristics unique to an individual perso
 
 ## Properties
 
+- identifier
+  - type: Email
```

Comment:
The property `identifier` is existing, of type `xsd:string` and is being used in `ExternalIdentifier` class.

Reply:
> In general, we do not follow an object-oriented model where we have "classes" that have "properties".

We are not doing object-oriented programming based on the concept of "objects" which contain data and code.

We are designing structured data, and JSON uses the word "object" for data values that have named "members", and for mapping types that map "keys" to "values".  So you are saying we aren't following a design approach where data "objects" have "keys" with "values"?  That doesn't sound scalable.

In the global describe-everything-in-the-universe ontology, common property names like "version" and "identifier" are first-come-first-served?  If SPDX decides that the "version" property must be a SemVer string, we've:
1) done a copyright search to ensure that nobody else has ever defined a "version" property with a different type, and 
2) can be confident that everybody else will follow the stake that we've claimed?

I'm not arguing that the model cannot have a set of common "properties" with names and values.  I am arguing that those property files are a library of name-value pairs that MAY be used, and not a set of constraints that MUST be followed.  Where "identifier" and "version" could have different types in different classes, the model could either refrain from defining identifier and value properties entirely (my recommendation), or define a property with one type while not prohibiting other types.